apps/admin/src/init/gmail.ts

```diff
@@ -0,0 +1,17 @@
+import type { IpcRenderer } from 'electron';
+import gmail from '@thecointech/tx-gmail';
+import { ConfigStore } from '@thecointech/store';
+// const { ipcRenderer } = window.require('electron');
+
+declare let window: Window & {
+  ipcRenderer: Pick<IpcRenderer, "invoke">
+};
+
+export async function initGmail() {
+  // open IPC bridge to node process.
+  gmail.bridge(window.ipcRenderer);
+  // Cycle the gmail token
+  const token = await ConfigStore.get("gmailcred")
```

Comment:
we store the token in the renderer thread to keep it within ConfigStore.
It just means one less file written to disk (this db is stored in electrons localStorage)